static/bidder-info/iqx.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://rtb.iqzone.com?pid={{.SourceId}}&host={{.Host}}&pbs=1"
+maintainer:
+  email: "it@iqzone.com"
```

Comment:
@IQzoneIT We have sent an email to it@iqzone.com to verify the maintainer email address as part of the onboarding process. Please revert back to that email with "Received" message to complete the verification.